### PR TITLE
Website updates

### PR DESCRIPTION
**Pull request by willgearty**
This groups the various functions in the website's reference pages. It also adds a link to the Palaeoverse twitter account.

Fixes https://github.com/palaeoverse-community/palaeoverse/issues/36.

### CODE REVIEW

File: _pkgdown.yml [hunk 1]
```diff
@@ -30,3 +30,62 @@ authors:
...
+
+home:
+  links:
+  - text: Twitter
+    href: https://twitter.com/ThePalaeoverse
+
+reference:
+- title: Data cleaning and preparing
+  desc: >
+    Various functions for cleaning and preparing your occurrence or taxonomic
```
Comment (LewisAJones) [inline _pkgdown.yml:42]:
```suggestion
    Functions for cleaning and preparing occurrence or taxonomic
```

File: _pkgdown.yml [hunk 2]
```diff
@@ -30,3 +30,62 @@ authors:
...
+    data before performing downstream analyses.
```
Comment (LewisAJones) [inline _pkgdown.yml:43]:
```suggestion
    data for downstream analyses.
```

File: _pkgdown.yml [hunk 3]
```diff
@@ -30,3 +30,62 @@ authors:
...
+  - phylo_check
+  - tax_unique
+  - palaeorotate
+  - look_up
+  - tax_expand_lat
+  - tax_expand_time
+
+- title: Data binning
+  desc: >
+    Various functions for binning occurrence data into spatial or temporal bins.
```
Comment (LewisAJones) [inline _pkgdown.yml:55]:
```suggestion
    Functions for binning occurrence data.
```

File: _pkgdown.yml [hunk 4]
```diff
@@ -30,3 +30,62 @@ authors:
...
+  contents:
+  - bin_lat
+  - bin_spatial
+  - bin_time
+
+- title: Data summarization
+  desc: >
+    Various functions to summarize occurrence-level data.
```
Comment (LewisAJones) [inline _pkgdown.yml:63]:
```suggestion
    Functions to summarise occurrence-level data.
```

File: _pkgdown.yml [hunk 5]
```diff
@@ -30,3 +30,62 @@ authors:
...
+
+- title: Data binning
+  desc: >
+    Various functions for binning occurrence data into spatial or temporal bins.
+  contents:
+  - bin_lat
+  - bin_spatial
+  - bin_time
+
+- title: Data summarization
```
Comment (LewisAJones) [inline _pkgdown.yml:61]:
```suggestion
- title: Data summarisation
```

File: _pkgdown.yml [hunk 6]
```diff
@@ -30,3 +30,62 @@ authors:
...
+  desc: >
+    Various functions to summarize occurrence-level data.
+  contents:
+  - tax_range_geo
+  - tax_range_time
+
+- title: Data visualization
+  desc: >
+    Functions to aid in the visualization of your data and the results of your analyses.
```
Comment (LewisAJones) [inline _pkgdown.yml:70]:
```suggestion
    Functions to aid in the visualisation of data and analysis results.
```

File: _pkgdown.yml [hunk 7]
```diff
@@ -30,3 +30,62 @@ authors:
...
+  contents:
+  - axis_geo
+
+- title: Auxiliary functions
+  desc: >
+    Various functions that help perform other functions.
```
Comment (LewisAJones) [inline _pkgdown.yml:76]:
```suggestion
    Functions to support further functionality.
```

**Review by LewisAJones — APPROVED** [review]
Hi Will, 
This looks fine to me and is good to go. I've suggested some changes to shorten the descriptions and to improve consistency ;)

File: _pkgdown.yml (hunk 5 above)
Comment (willgearty) [reply _pkgdown.yml:61]: :satisfied:

**Pull request merged**